OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -50,6 +50,17 @@ public interface INotifySold
 		void Sold(Actor self);
 	}
 
+	[RequireExplicitImplementation]
+	public interface INotifyChangedCustomLayer
+	{
+		void JumpjetLaunch(Actor self);
```

Comment:
Can we please simplify this down to a single `CustomLayerChanged(CustomMovementLayerType old, CustomMovementLayerType new)`?